src/main/perf/LineFileDocs.java

```diff
@@ -520,6 +561,9 @@ public Document nextDoc(DocState doc) throws IOException {
       doc.dateCal.setTime(date);
       msecSinceEpoch = doc.dateCal.getTimeInMillis();
       timeSec = doc.dateCal.get(Calendar.HOUR_OF_DAY)*3600 + doc.dateCal.get(Calendar.MINUTE)*60 + doc.dateCal.get(Calendar.SECOND);
+      if (doc.vector != null) {
+        doc.vector.setVectorValue(lfd.vector);
```

Comment:
Cool!   So we can re-use the same `VectorField` (but it is a new `float[]` for each doc).